eng/scripts/splittestdependencies/tests/inputs/projects.txt

```diff
@@ -41,6 +41,9 @@ $(RepoRoot)sdk\core\Azure.Core.Experimental\tests\perf\Azure.Core.Experimental.P
 $(RepoRoot)sdk\core\Azure.Core.TestFramework\tests\Azure.Core.TestFramework.Tests.csproj
 $(RepoRoot)sdk\core\Azure.Core\tests\Azure.Core.Tests.csproj
 $(RepoRoot)sdk\core\Azure.Core\tests\public\Azure.Core.Tests.Public.csproj
+$(RepoRoot)sdk\core\System.Net.ClientModel\tests\System.Net.ClientModel.Tests.csproj
+$(RepoRoot)sdk\core\System.Net.ClientModel\tests\clients\System.Net.ClientModel.Tests.Clients.csproj
+$(RepoRoot)sdk\core\System.Net.ClientModel\tests\internal\System.Net.ClientModel.Tests.Internal.csproj
```

Comment:
This should make it so that test failures block CIs now.

Reply:
This should not be needed if you apply the csproj flags I mentioned below.  My pr yesterday had these flags and I added broken tests to both projects and both were executed and caught in ci.